### PR TITLE
Rework test cloud

### PR DESCRIPTION
**Pull request by heavenboy8**
- Rewrite test for `cloud` (as a mirror of `home`)
- Finally make the tests pass
- Fixing new linting issue (due to increase deps)

### CODE REVIEW

File: documentation/docs/cloud/configuration.md [hunk 1]
```diff
@@ -9,37 +9,42 @@ tests_cmd = "pytest"
...
+| Keys  | Required | Types  |                  Description                  |
+| :---: | :------: | :----: | :-------------------------------------------: |
+| code  |    ✔️     | string | Relative path to the application code folder  |
+| tests |    ✔️     | string | Relative path to the application tests folder |
+
 |     docker      |    ✔️     |          string          | URL to the mse docker to run. It could be a local docker to run local test but it must be a remote url when deploying. See [below section](./configuration.md#mse-docker) |
 |     project     |    ✔️     |          string          |                                                             Project name to regroup applications for payment                                                              |
 |    hardware     |    ✔️     |          string          |                                                           Name of the hardware booked to spawn your application                                                           |
```
Comment (ThibsG) [inline documentation/docs/cloud/configuration.md:50]: formatting

File: mse_cli/core/conf.py [hunk 2]
```diff
@@ -290,4 +294,4 @@ def into_cloud_payload(self) -> Dict[str, Any]:
             "domain_name": cloud.ssl.domain_name if cloud.ssl else None,
             "hardware": cloud.hardware,
             "docker": cloud.docker,
-        }  # Do not send the private_key or the code location
+        }  # Do not send the private_key or the code code
```
Comment (ThibsG) [inline mse_cli/core/conf.py:301]: `code location` (sorry, cannot add suggestion here)

File: documentation/docs/cloud/subcommand/context.md [hunk 3]
```diff
@@ -80,6 +86,9 @@ This directory contains:
     name = "helloworld"
     project = "default"
     python_application = "app:app"
+    tests = "/home/user/tests"
```
[the remark this answers is not in the document]
Comment (heavenboy8) [reply documentation/docs/cloud/subcommand/context.md:89]: The context file is generated by the cli. If the user has set `tests = "tests"` in the app conf, the cli will expand it to get the absolute path. The absolute path is then stored in the context. I don't now the name of the user, so I put that.

File: mse_cli/core/conf.py (hunk 2 above)
Comment (heavenboy8) [reply mse_cli/core/conf.py:301]: f**k

File: documentation/docs/cloud/configuration.md (hunk 1 above)
Comment (heavenboy8) [reply documentation/docs/cloud/configuration.md:50]: good catch. an extra useless \n

**Pull request merged**